db/seeds.rb

```diff
@@ -11,7 +11,7 @@
 
 templates.each do |template|
   template[:template] = File.read(File.join(ForemanChef::Engine.root, "app/views/foreman/unattended", template.delete(:source)))
-  ProvisioningTemplate.find_or_create_by_name(template).update_attributes(defaults.merge(template))
+  ProvisioningTemplate.where(:name => template).first_or_create.update_attributes(defaults.merge(template))
```

Comment:
this doesn't work for me:
`
PG::Error: ERROR:  missing FROM-clause entry for table "name"
LINE 1: ...templates"."type" IN ('ProvisioningTemplate') AND "name"."na...
`
using `template[:name]` instead of `template` fixed it.